packages/uicore/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.91.2",
+  "version": "3.91.3",
```

Since this is exposing new optional functionality, it isn't a bug fix, so should be a point release, not a patch.
```suggestion
  "version": "3.92.0",
```